### ISSUE TITLE
[COMPILER REQUEST]: Add support for Tricore GCC

### ISSUE THREAD

**Issue by jferreir** [compiler-explorer/compiler-explorer#6321]
### Compiler name

Tricore GCC

### Compiler version

11.3.1

### Compiler language

C++, Assembly and C

### Compiler homepage

https://github.com/EEESlab/tricore-gcc-toolchain-11.3.0

### Motivation

The Tricore architecture is one of the most used architecture for real time control and safety related applications in automotive 
Please check https://github.com/compiler-explorer/misc-builder/pull/80

**Comment by dkm** [comment]
Hmmm, would have been nice to have it along all the other gcc build we have... Have you tried to build it using ct-ng? If not, maybe I can try that. That would make our maintenance easier

**Comment by dkm** [comment]
Yes, tricore is very interesting, too bad it never made it upstream :(

**Comment by dkm** [comment]
I'll revive this one by setting up the ct-ng config + the usual update of zillions config files.

**Comment by dkm** [comment]
This is now live 🚀 thanks @cpackham and @jferreir . cc @gtagliavini
